torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -67,10 +67,16 @@ namespace jit {
 namespace tensorexpr {
 
 std::string buildErrorMessage(const std::string& s) {
-  // TODO: Update this generic error message to include details regarding
-  // turning off the fuser.
-  static const std::string generic_error_message = "";
-  return s + " " + generic_error_message;
+  static const std::string generic_error_message =
+      "This error occured in the fuser. You can turn off the fuser by setting "
+      "torch._C._jit_override_can_fuse_on_cpu to false";
```

Comment:
Maybe "You can turn off the fuser with torch._C._jit_override_can_fuse_on_cpu(False)"
Although it's kind of annoying if it's the GPU fuser but 🤷

Reply:
Updated that. 

Good point regarding GPU fuser. Should we provide one flag to turn both off?